sequencer/src/api/data_source.rs

```diff
@@ -141,36 +145,109 @@ pub(crate) trait CatchupDataSource {
 
 impl CatchupDataSource for MetricsDataSource {}
 
-pub struct MyHotShotConfig(Value);
-
-impl MyHotShotConfig {
-    pub fn new(c: HotShotConfig<PubKey>) -> anyhow::Result<Self> {
-        let mut value = serde_json::to_value(c)?;
-
-        let validator_config = value
-            .get_mut("my_own_validator_config")
-            .context("my_own_validator_config not found")?;
-
-        let private_key = validator_config
-            .get_mut("private_key")
-            .context("private_key not found")?;
-        *private_key = "******".into();
+#[derive(Debug, Serialize)]
+pub struct PublicValidatorConfig {
+    pub public_key: PubKey,
+    pub stake_value: u64,
+    pub is_da: bool,
+    pub private_key: &'static str,
+    pub state_key_pair: &'static str,
```

Comment:
We can at least display the public state key (`state_key_pair.ver_key()`) in plaintext